### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by HectorTorresE**
add the readme file, the index.html with an h1 header also add style.css with the header class, and add the gitignore with the test.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,72 @@
...
...y6oJHXz1l54LgkJNOwJMaLoDIKehK8xhkZvs7jRQLf/9ciqkbX0nW13F0Ny81meRlax+XlcHdthuxtyryhOGMJhEnU6G9WdUEP/4+hoT8Wi6MvGrV33eEeJ/S2XtFXvt2rCre5ha3XZkRYsUOG0Ac+BHlGa4DxxLHFtpLGl6Ev30p8SC/Ngyps5EUvlh5Mi+xDLNm5/o4cKWyO4Ev4FF5iIdb4KHHw8uaeT+8aLS6YiLIbBAaJbHsO5WvGGUIOvgK6E/TMuB6kh+ZxYyqWuB4VIxaL9QoWOPv2hMB/55kQJWNLSnGWHJ0XQHun79OJlmx3oRzyJic623IGrcFxSbHxgAUq3nJ8Ty0/t5CTpoTz8K8Sq5YtepUij/LsBvS4lCU9fVT3UGJeFk9VQpGq90WN0csy1T/PwT8BQie+oYshW3lmFHFockEKGTr0DNS3sr0NOCmJtUrHCsbK4KoJrXq8cK3fKwZVuFudHOHkOz6Lj5OvKSybpHmPR/qOlBD+rDF8N/uKlPCudLIqePBdHa9vlRVrV1CIL6zBAwTmshiS+cz3o4uR6HTxOpYa6JyQ5ddzAtyB2F75soEdwcRM3X7lYzRqjyb51DGlV7eD35LWidVdP96PGbCfSkxOsPs0vVPT+4c4FC5C0CJBCvt3ejya3P5SoXl20sBkXv7CQVbFyiqynqs6k37SI6TOxihdP519qxZfqmyv4apxztjw9S+8KjbG+zNlrqpOKKkU/eBVWaWMokt4ZmnHxux2chmHI82NYhtqlGi5M+Ku4xb4uzyL+PiCdkI3Jj5ArMYXu4iiZTbx0FQ5QDpeeKX5HIGNj+rYrSJRrDdu3/o7WZgfbk1f9gDMRsirO2sX3+JuFLwAr0Vb7nQC+Stg6Ra74DY7DNbqCrtS4XWsII8WsYiP+PnB9f9pdoUQTb2kUHfxvX5bTawwy2Frjkj2e7uJZzd+KKE3iDP2LchXb2KM/MHXlOdshv3SP/g1Rhl6+ZdebuDnT06hMw4b8lKto7FZ/bXoHlJhux4PJnS7ztpHN9aE/2O/KnYYSu/16fH1b+6u1qsGxqxDp/wpQRjgnbLLsSeuLXXxOxjAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzAMwzC/gf8ArWupqIu2W5AAAAAASUVORK5CYII=)
```
Comment (Deepakdanger) [inline README.md]: - [ ] The Screenshot picture is not displaying. It redirects to a new link to the same repo. Kindly Fix it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,72 @@
...
+
+## Built With
+
+- HTML and CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```
Comment (Deepakdanger) [inline README.md]: - [OPTIONAL ] 
- [ ] This live demo link is not working . Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users

File: README.md [hunk 3]
```diff
@@ -1,2 +1,72 @@
...
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/HectorTorresE)
+- Twitter: [@twitterhandle](https://twitter.com/HectorT00406915)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
```
Comment (Deepakdanger) [inline README.md]: - [ ] There is no need for Author2. Please remove it, It's taking unnecessary attention.

File: test.md [hunk 4]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (Deepakdanger) [inline test.md]: - [x] This file is ignored by `.gitignore`. Kindly Fix the path for this file in `.gitignore`.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi, @HectorTorresE, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Good PR ✔️ 
- All Linter is passinng ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```
Comment (mateo951) [inline .gitignore:2]: Please be sure to remove the test.md file from your remote repository 🙌 Well done applying all required changes by the previous reviewer!

**Review by mateo951 — APPROVED** [review]
Hi @HectorTorresE, 

Nice work on making all the necessary changes requested by the previous code reviewer!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**